core/templates/dev/head/pages/community-dashboard-page/voiceover-opportunities/voiceover-opportunities.directive.html

```diff
@@ -1,5 +1,5 @@
 <div>
-  <opportunities-list data-loading="$ctrl.opportunitiesAreLoading"
+  <opportunities-list loading-view="$ctrl.opportunitiesAreLoading"
```

Comment:
The name loading-view seems odd. It seems to be a boolean? But the name doesn't imply a boolean.